--- devfsd/GNUmakefile.orig	Mon May 26 21:18:17 2003
+++ devfsd/GNUmakefile	Mon May 26 21:18:37 2003
@@ -20,10 +20,10 @@
 
 OBJECTS = devfsd.o expression.o compat_name.o
 
-CFLAGS = -O2 -I. -I$(KERNEL_DIR)/include -Wall $(CEXTRAS) $(DEFINES)
+CFLAGS = -I. -I$(KERNEL_DIR)/include -Wall $(CEXTRAS) $(DEFINES)
 
 devfsd:	$(OBJECTS)
-	$(CC) -O2 -o devfsd $(OBJECTS) $(LDFLAGS) -export-dynamic -ldl
+	$(CC) -o devfsd $(OBJECTS) $(LDFLAGS) -export-dynamic -ldl
 
 install: devfsd
 	@if fgrep -q "Generic section: do not change" /etc/modules.conf ; then\
