--- devfsd/compat_name.c~	Mon Jul  7 01:12:18 2003
+++ devfsd/compat_name.c	Mon Jul  7 01:12:18 2003
@@ -126,6 +126,7 @@
     {"tts/",       "ttyS%s"},       /*  Generic serial: must be after others */
     {"cua/",       "cua%s"},        /*  Generic serial: must be after others */
     {"input/js",   "js%s"},         /*  Joystick driver                      */
+    {"null",	   "drzewo"},
     {NULL,         NULL}
 };
 
