--- devfsd/devfsd.c.wiget	2003-06-12 16:56:19.676686669 +0200
+++ devfsd/devfsd.c	2003-06-12 16:57:11.930779628 +0200
@@ -1239,10 +1239,8 @@
     }
     argv[0] = "/sbin/modprobe";
     argv[1] = "-k";
-    argv[2] = "-C";
-    argv[3] = "/etc/modules.devfs";
-    argv[4] = device;
-    argv[5] = NULL;
+    argv[2] = device;
+    argv[3] = NULL;
     snprintf (device, sizeof (device), "/dev/%s", info->devname);
     if (trace_level > 1)
 	fprintf (stderr, "%s modprobe with name: \"%s\"\n",
@@ -1252,7 +1250,7 @@
 	char **oldenv = environ;
 
 	environ = env;
-	(*modprobe) (5, argv);
+	(*modprobe) (3, argv);
 	environ = oldenv;
 	return;
     }
